Resources/Prototypes/Catalog/Cargo/cargo_vending.yml

```diff
@@ -33,8 +33,8 @@
     sprite: Objects/Specific/Service/vending_machine_restock.rsi
     state: base
   product: CrateVendingMachineRestockClothesFilled
-  cost: 7250 #DeltaV
-  category: Service
+  cost: 7245
```

Comment:
okay i don't wanna be *that* guy but this should be 7250 or something 😢

Reply:
Infinite money exploit obtained, repeatedly buying and reselling clothesmate stocks for 5 spesos each lmao.

Reply:
buy and sell prices are the same? that's insane.

Reply:
put it back on 7245

Reply:
Lol. Yea it's to prevent people from doing cargo exploits.